src/notification.test.ts

```diff
@@ -392,9 +428,14 @@ describe("BaseNotification", () => {
         fetch: fetchFn,
       });
 
-      await expect(
-        notification.fetchProtocolNegotiationInfo(),
-      ).rejects.toThrow();
+      const err = await notification
+        .fetchProtocolNegotiationInfo()
+        .catch((e) => e as UnauthorizedError);
+
+      expect(err).toBeInstanceOf(UnauthorizedError);
+      expect((err as UnauthorizedError).problemDetails.status).toBe(401);
+      expect((err as UnauthorizedError).problemDetails.title).toBeFalsy();
```

Comment:
I would be more specific on this test, by either setting a value for it in the mocked response `statusText` and checking for it here, or by either not including this check at all, or checking specifically for `""`, which is the default value. I'm not usually fond of checking for defaults that are not part of our API, so I would lean towards setting a `statusText` or not including this check at all.

Reply:
Addressed with https://github.com/inrupt/solid-client-notifications-js/pull/1031/commits/72a22a61996e577f433700f249841514b4cb4674